runtime/jnichk/jnicwrappers.c

```diff
@@ -4091,6 +4090,57 @@ checkGetObjectRefType(JNIEnv *env, jobject obj)
 	return actualResult;
 }
 
+#if JAVA_SPEC_VERSION >= 9
+static jobject JNICALL
+checkGetModule(JNIEnv *env, jclass clazz)
+{
+	jobject actualResult = NULL;
+	J9JniCheckLocalRefState refTracking;
+	static const U_32 argDescriptor[] = { JNIC_JCLASS, 0 };
+	static const char function[] = "GetModule";
+
+	jniCheckArgs(function, 0, CRITICAL_WARN, &refTracking, argDescriptor, env, clazz);
+	actualResult = globalJavaVM->EsJNIFunctions->GetModule(env, clazz);
+	jniCheckFlushJNICache(env);
+
+	return actualResult;
+}
+#endif /* JAVA_SPEC_VERSION >= 9 */
+
+#if JAVA_SPEC_VERSION >= 19
+static jboolean JNICALL
+checkIsVirtualThread(JNIEnv *env, jobject obj)
+{
+	jboolean actualResult = JNI_FALSE;
+	J9JniCheckLocalRefState refTracking;
+	static const U_32 argDescriptor[] = { JNIC_JOBJECT, 0 };
+	static const char function[] = "IsVirtualThread";
+
+	jniCheckArgs(function, 0, CRITICAL_WARN, &refTracking, argDescriptor, env, obj);
+	actualResult = globalJavaVM->EsJNIFunctions->IsVirtualThread(env, obj);
+	jniCheckFlushJNICache(env);
```

Comment:
Shouldn't this call jniCheckLocalRefTracking()?

Reply:
I'll add a call here and in `checkGetModule()`, but it's not clear to me that they're necessary for functions which can't return a reference (e.g. I don't think they're necessary in `checkGetStringUTFLength()` or `checkGetStringUTFLengthAsLong()`).

Reply:
I created https://github.com/eclipse-openj9/openj9/issues/20478 to follow up on uses of `jniCheckLocalRefTracking()`.